dlt/dlt_broker.py

```diff
@@ -85,21 +93,26 @@ def remove_context(self, context_queue):
 
     def stop(self):
         """Stop the broker"""
-        logger.info("Stopping DLTContextHandler and DLTMessageHandler")
+        logger.info("Stopping DLTContextHandler, DLTTimeHandler and DLTMessageHandler")
```

Comment:
There is no more DLTTimeHandler, please remove here.

Reply:
done